web/channels/player_channel.ex

```diff
@@ -6,7 +6,7 @@ defmodule Yggdrasil.PlayerChannel do
   require Logger
 
   def join("player:" <> user_id = topic, _message, socket) do
-    if socket.assigns.user == String.to_integer(user_id) do
+    if socket.assigns.user == user_id do
```

Comment:
It's a string when guardian pulls it out, but I suppose it could be converted to an integer early on and put into assigns as an int. The change to guardian is why this happened and it's [here](https://github.com/voluspa/yggdrasil/blob/json.api/web/channels/user_socket.ex#L28).

Sort of took the stance if it's coming out as string and you are having to parse to an int each time that it for now maybe best to just leave it as a string and not have to convert each time.